css/base.css

```diff
@@ -35,13 +35,25 @@ body {
 }
 
 .main-nav ul {
-  display: none;
+  /*display: none;*/
 }
 
 .main-nav ul li {
```

Comment:
I cloned your repo to play with your site. I want to spend more time tonight working with your main nav section and hover. I'll report back here.